### PR TITLE
Make rubber rounds research exclusive, but provide disablers.

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

This PR removed the rubber rounds from security spawn loadouts, but provides them disablers instead. I decided to try this first after chewing on what position rubber bullets vs disablers play in the games combat, and after some discussions about whether rubber bullets are different enough from disablers or too difficult to balance using the same weapons as lethal rounds. 

I think trying this first before anything more drastic is a good idea since it accomplishes two things. It makes the rubbers much less common and more of a research goal to match their effect in game of an upgrade to security's disabler tech. It also allows us to keep the entities in game while we wait on new med, which I am hopeful will give us some options to differentiate the two non-lethal techs with more less-than-lethal levers to pull like pain etc.

## Why / Balance

Rubber bullets are in a weird spot currently. They're given out at round start while disablers are not, yet they are often more effective than disablers. This fixes that. They are also difficult to balance because they need to be considered along-side the use of lethal rounds in the same weapons. This kicks that can down the road slightly.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Security is no longer issued rubber bullets at round start.
- tweak: Security is now issued Disablers at round start.

### CODE REVIEW

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@
     id: DetectivePDA
     ears: ClothingHeadsetSecurity
     belt: ClothingBeltHolsterFilled
+    pcoket1: WeaponDisabler
   innerClothingSkirt: ClothingUniformJumpskirtDetective
```
Comment (metalgearsloth) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:37]: did you check this

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Don't make sec start with even more bloat, just put it in vendors, they don't need a whole armoury on them.

This doesn't really do anything to address the rubber problem either.

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml (hunk 1 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Roles/Jobs/Security/detective.yml:37]: sheeeit.

**Comment by EmoGarbage404** [comment]
> This kicks that can down the road slightly

go the mile and yeet it entirely.

**Comment by IProduceWidgets** [comment]
I'm happy to do that, I just kind of expect us to want them once we have mew med stuff to play with which will actually differentiate them. 

I could yeet the research for now, and then they wouldn't be acquirable. Would that be sensible? I guess we can always revert this PR later... hmmm.

**Comment by Nairodian** [comment]
Shit, I just finished a PR with all that. Should I just PR that or leave it to you?

**Comment by IProduceWidgets** [comment]
> Shit, I just finished a PR with all that. Should I just PR that or leave it to you?

Oh if you've got it, by all means, I can close.

**Comment by Nairodian** [comment]
#26470

**Pull request closed without merging**